lib/App/Sqitch.pm

```diff
@@ -176,6 +176,17 @@ has pager => (
     },
 );
 
+has strict => (
```

Comment:
Since the option applies only to deploys and reverts, IIUC, I think it'd be better placed in `RevertDeployCommand.pm`. Would require updating `around options` and `around configure` in that file.

Reply:
It also applies to just `revert`, which doesn't inherit from `RevertDeployCommand`, or at least that's how I understand it.

If this were in `RevertDeployCommand`, could we make the change to `Engine.pm` in this PR?

Reply:
Right, but it's not an option for all or even most commands. A similar pattern exists for `no_prompt` and `prompt_accept`: They're defined both in RevertDeployCommand and separately in revert.pm. I think we should stick to that pattern and not make it a global option.